spec/controllers/orphans_controller_spec.rb

```diff
@@ -8,12 +8,55 @@
   before { create(:user, login: "orphan_account") }
 
   let!(:user) { create(:user) }
-
   let!(:pseud) { create(:pseud, user: user) }
   let!(:work) { create(:work, authors: [pseud]) }
   let(:second_work) { create(:work, authors: user.pseuds) }
   let(:series) { create(:series, works: [work], authors: [pseud]) }
 
+  let!(:suspended_user) { create(:user, suspended: true, suspended_until: 1.week.since) }
+  let!(:suspended_pseud) { create(:pseud, user: suspended_user) }
```

Comment:
Could this be the default pseud?

Reply:
Well the original tests have `let!(:pseud) { create(:pseud, user: user) }`, so I just replicated that for the suspended version. I don't know if there are any complications with not being able to orphan a default pseud or something like that, so I think that sort of refactoring is probably better left to another separate task.